jsapp/js/projects/projectViews.module.scss

```diff
@@ -19,3 +20,25 @@
   flex: 1;
   justify-content: flex-end;
 }
+
+.banner {
+  display: flex;
+  justify-content: space-between;
+  margin: sizes.$x24;
```

Comment:
I haven't seen the designs, but is bottom margin 24px needed? There is already 30px from `projectViews-module__header`. Just need confirmation this is what we want :)

Reply:
Removed the bottom margin :)